### DISCUSSION TITLE
Gregorian holidays - always INVALID

### DISCUSSION THREAD

**Discussion by zdunex25** [zepp-health/zeppos-forum#134] [Developer Feedback]
Documentation says we should be able display **Gregorian Holidays** on the Watchface: [timegetshowfestival](https://docs.zepp.com/docs/watchface/api/hmSensor/sensorId/TIME/#timegetshowfestival) however it returns _INVALID_.
I was able to find this under App development:
![image](https://user-images.githubusercontent.com/1674725/230604956-bdd6c245-d450-4574-8bc2-7f18dbf81b27.png)

Today for example is **Good Friday** so it should be able to show it, but it does not. What is the problem here?
@MyoungXUE

**Comment by MyoungXUE** [comment]
Hi, this is a documentation error, this API only works when the language is set to Chinese, we will fix it as soon as possible.

**Reply by MyoungXUE** [reply]
The document has been updated, please refresh and check again.
